package.json

```diff
@@ -373,9 +368,12 @@
     "@microsoft/fetch-event-source": "^2.0.1",
     "@opentelemetry/api": "^1.1.0",
     "@opentelemetry/context-zone": "^1.5.0",
+    "@opentelemetry/core": "^1.7.0",
     "@opentelemetry/exporter-trace-otlp-http": "^0.31.0",
     "@opentelemetry/instrumentation": "^0.31.0",
     "@opentelemetry/instrumentation-fetch": "^0.31.0",
+    "@opentelemetry/resources": "^1.7.0",
+    "@opentelemetry/sdk-trace-base": "^1.7.0",
```

Comment:
If I added some new deps here, it's because they were being depended upon directly but were actually transitive deps before pnpm.